src/components/modeler/Modeler.vue

```diff
@@ -847,6 +945,27 @@ export default {
         });
       });
     },
+    async addClonedNodes(nodes) {
+      nodes.forEach(node => {
+        if (!node.pool) {
+          node.pool = this.poolTarget;
+        }
+  
+        const targetProcess = node.getTargetProcess(this.processes, this.processNode);
+        addNodeToProcess(node, targetProcess);
+  
+        this.planeElements.push(node.diagram);
+        store.commit('addNode', node);
+        this.poolTarget = null;
+
+        return new Promise(resolve => {
+          setTimeout(() => {
+            this.pushToUndoStack();
+            resolve();
+          });
+        });
+      });
```

Comment:
this can be 
```suggestion
        await this.pushToUndoStack();
      });
```
since I believe pushToUndoStack() is also an async function

Reply:
although, the setTimeout thing.... don't know how to deal with that. I'm guessing the setTimeout is because of the `nextThick` thing... Maybe do 
```suggestion
await this.$nextThick(); // not sure the naming convention here
await this.pushToUndoStack();
```